keras/layers/convolutional.py

```diff
@@ -3514,6 +3514,11 @@ def compute_output_shape(self, input_shape):
   def call(self, inputs):
     # pylint: disable=invalid-unary-operand-type
     if self.data_format == 'channels_first':
+      if ((inputs.shape[2] is not None and sum(self.cropping[0]) >= inputs.shape[2]) or
+          (inputs.shape[3] is not None and sum(self.cropping[1]) >= inputs.shape[3])):
+          raise ValueError('cropping parameter of Cropping layer must be '
```

Comment:
"Argument `cropping`" (with backticks) 

"of Cropping layer" is not necessary, the traceback will show the origin